app/src/main/java/cm/aptoide/pt/appview/RelatedContentView.kt

```diff
@@ -40,15 +38,17 @@ import cm.aptoide.pt.feature_reactions.ui.ReactionsView
 fun RelatedContentView(
   packageName: String,
   onRelatedContentClick: (String) -> Unit,
-  listScope: LazyListScope?,
+
 ) {
   val editorialsMetaViewModel = relatedEditorialsCardViewModel(packageName = packageName)
   val uiState by editorialsMetaViewModel.uiState.collectAsState()
-
-  listScope?.item { Box(modifier = Modifier.padding(top = 24.dp)) }
-  listScope?.items(uiState ?: emptyList()) { editorialMeta ->
-    RelatedContentCard(editorialMeta, onRelatedContentClick)
+  Column {
+    Box(modifier = Modifier.padding(top = 24.dp))
```

Comment:
This padding can be applied to the parent `Column`, instead of creating a new `Box`. An alternative would also be to use `Spacer` instead of `Box`